lumicks/pylake/force_calibration/detail/driving_input.py

```diff
@@ -101,7 +101,9 @@ def estimate_driving_input_parameters(
     return amp, freq
 
 
-def response_power_density(psd_data, sample_rate, driving_frequency, num_windows, freq_window=50.0):
+def power_density_driving_frequency(
```

Comment:
I'm not sure about this new name, it took me quite a while that it should read more like "power density -and- driving frequency" instead of just one thing .... I guess maybe `power_density_and_driving_frequency` ? `input_power_and_frequency`? neither are particularly nice, but I do think it's clearer.

Reply:
I discussed it with Ray a bit. We decided to go with `driving_power_peak`.